brambling/utils/model_tables.py

```diff
@@ -615,7 +638,7 @@ def get_field_val(self, obj, key):
             field = "spaces"
 
         if date_str:
-            if obj.get_eventhousing():
+            if obj.get_eventhousing() and obj.providing_housing:
```

Comment:
hmm. yeah, that kinda makes sense... because it doesn't try the date_str path, it tries the default: getting the attribute.